cost/aws/unattached_volumes/aws_delete_unattached_volumes.pt

```diff
@@ -0,0 +1,469 @@
+    startDate.setDate(startDate.getDate() - unattachedDays);
+    var startDateInEpochTime = startDate.getTime() / 1000;
+    logs = {
+      auth: "auth_aws",
+      host: 'cloudtrail.'+region+'.amazonaws.com',
+      path: '/',
+      verb: 'POST',
+      pagination: "aws_cloudtrails_pagination_xml",
+      headers: {
+        "X-Amz-Target": "com.amazonaws.cloudtrail.v20131101.CloudTrail_20131101.LookupEvents",
+        "Content-Type": "application/x-amz-json-1.1",
+      },
+      body_fields:{
+        "StartTime" : startDateInEpochTime,
+        "LookupAttributes" :[{
+          "AttributeKey": "ResourceName",
+          "AttributeValue": volume_id
+        }],
+        "EndTime" : endDateInEpochTime
+      }
+    }
+
+  // To Delay Next Request
+  // Due to AWS ThrottlingException (Rate Limit)
+  // Delaying the next request for 1 second.
+  var current_time = Math.round((new Date()).getTime())
+  var endTime   = current_time + ( 1000)
+  while (current_time <= endTime) {
+    current_time = Math.round((new Date()).getTime())
+  }
+  EOS
+end
+
+script "js_filter_volumes_unattached_for_long", type:"javascript" do
+  parameters "cloud_trail_logs","unattached_volumes","unattached_days"
+  result "trail_logs"
+  code <<-EOS
+  var trail_logs = [];
+  for(var i =0 ; i < unattached_volumes.length; i++){
+    vol = unattached_volumes[i];
+    for(var j=0; j < cloud_trail_logs.length; j++){
+      log = cloud_trail_logs[j];
+      // Check, if the volume has been unattached for more than param_unattached_days, ignore the volume if the unattached for less than param_unattached_days
+      if(vol['vol_id'] === log['volume_id']){
+        if(log['eventName'] === 'DetachVolume' || log['eventName'] === 'CreateVolume'){
+          var eventOccurDate = new Date(log['eventTime'] * 1000);
+          var today = new Date();
+          var time_difference = today.getTime() - eventOccurDate.getTime();
+          var days_difference = time_difference / (1000 * 3600 * 24);
+            if(days_difference <= unattached_days){
+              trail_logs.push({
+                volume_id : vol['vol_id'],
+                region : log['region'],
+                availability_zone : log['availability_zone'],
+                tagKeyValue : vol['tagKeyValue'],
+                eventName : log['eventName'],
+                volume_Type : vol['volume_Type']
+                eventTime : (eventOccurDate).toString()
+              })
+            }
+          break;
+        }
+      }
+    }
+  }
+  trail_logs = _.sortBy(trail_logs, 'region');
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_unattached_volumes_list" do
+  validate $ds_unattached_volumes_for_long do
+  summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} UnAttached Volumes Found in AWS"
```

Comment:
should be UnAttached

Reply:
Hi, @cdwilhelm  where I need to change exactly?

Reply:
change UnAttached to Unattached.  line 302